share/palettes/CoolWarm.tf3

```diff
@@ -1,31 +1,16 @@
-<?xml version="1.0" encoding="ISO-8859-1" standalone="yes"?>
 <MapperFunctionParams>
 <OpacityComposition Type="Long">
   0 
 </OpacityComposition>
 <DataBounds Type="Double">
-  -1 1 
+  0 1 
 </DataBounds>
 <OpacityScale Type="Double">
   1 
 </OpacityScale>
-<ColorMapParams>
```

Comment:
I did not change the colormap file format. None of the current divergent colormaps specified whether they used whitepoints or not which they should. This was being implicitly set to use whitepoint in the getter function. Since there is no way to set a default value in the params database without writing a getter/setter I made sure the colormaps specified this. I also fixed some of the default control points which were at odd intervals.

Reply:
One thing to clarify is that the _init() functions allow you to specify initial values when creating a new param but not default values.

Reply:
Sounds good.